src/hooks/useFetch.js

```diff
@@ -0,0 +1,46 @@
+import { useCallback } from 'react';
+
+const useFetch = () => {
```

Comment:
"It will be a logger, which logs api calls and save to a local storage"

It should also log request to a console, let it be future change